File: C/basic_arithmetic.c
#include <stdio.h>
int main() {
    int a, b;
    printf("Enter two integers: ");
    scanf("%d %d", &a, &b);

    int sum = a + b;
    int difference = a - b;
    int product = a * b;
    float quotient = (b != 0) ? (float)a / b : 0;

    printf("Sum: %d\n", sum);
    printf("Difference: %d\n", difference);
    printf("Product: %d\n", product);
    if (b != 0) {
        printf("Quotient: %.2f\n", quotient);
    } else {
        printf("Quotient: Division by zero is undefined.\n");
    }

    return 0;
}

File: Python/basic_arithmetic.py
def main():
    a_b = input("Enter two integers: ").split()
    a = int(a_b[0])
    b = int(a_b[1])

    sum_ = a + b
    difference = a - b
    product = a * b
    quotient = (a / b) if b != 0 else 0.0

    print(f"Sum: {sum_}")
    print(f"Difference: {difference}")
    print(f"Product: {product}")

    if b != 0:
        print(f"Quotient: {quotient:.2f}")
    else:
        print("Quotient: Division by zero is undefined.")


if __name__ == "__main__":
    main()

